### PR TITLE
Migrate to the UMB

### PR DESCRIPTION
**Pull request by mprahl**
@ralphbean, as part of the review, do you mind double checking that my fake messages match how Moksha presents them? I copied the messages from datagrepper and then modified them to look like how they'd come when processed by Moksha but I'm concerned I could have made a mistake along the way.

### CODE REVIEW

**Comment by ralphbean** [comment]
Exciting!  I'll have a look tomorrow.  Thanks @mprahl!

File: resultsdbupdater/consumer.py [hunk 1]
```diff
@@ -18,15 +22,20 @@ def debug_log_msg(self, msg):
...
-            elif 'ci_type' in msg['headers']:
-                if msg['headers']['ci_type'] == 'ci-tps':
-                    self.debug_log_msg(msg)
-                    return tps_post_to_resultsdb(msg)
+        resultsdb_style_topics = [
+            '/topic/VirtualTopic.eng.platformci.covscan.result',
+            '/topic/VirtualTopic.qe.ci.jenkins'
+        ]
+        if msg['topic'].startswith(
+                '/topic/VirtualTopic.eng.platformci.rpmdiff'):
```
Comment (ralphbean) [inline resultsdbupdater/consumer.py]: Any reason not to add this to `resultsdb_style_topics`?

File: tests/fake_messages/message.json [hunk 2]
```diff
@@ -1,39 +1,53 @@
...
-            "tests" : [
-                { "executor" : "beaker" , "arch" : "aarch64" , "executed" : "25", "failed" : "0" },
-                { "executor" : "CI-OSP" , "arch" : "x86_64" , "executed" : "1000",   "failed" : "0" }
-            ],
-            "trigger" : "git push"
-        },
-        "topic": "/topic/CI"
-    },
+  "body": {
     "headers": {
```
Comment (ralphbean) [inline tests/fake_messages/message.json]: OK.  I *think* there are not headers *inside* the body.  That is probably an artifact introduced by datagrepper.  I think when the messages show up to `consume` they only appear *alongside* the body.

File: resultsdbupdater/consumer.py [hunk 3]
```diff
@@ -18,15 +22,20 @@ def debug_log_msg(self, msg):
...
-                elif msg['headers']['CI_TYPE'] == 'resultsdb':
-                    self.debug_log_msg(msg)
-                    return resultsdb_post_to_resultsdb(msg)
-            elif 'ci_type' in msg['headers']:
-                if msg['headers']['ci_type'] == 'ci-tps':
-                    self.debug_log_msg(msg)
-                    return tps_post_to_resultsdb(msg)
+        resultsdb_style_topics = [
+            '/topic/VirtualTopic.eng.platformci.covscan.result',
+            '/topic/VirtualTopic.qe.ci.jenkins'
```
Comment (thrix) [inline resultsdbupdater/consumer.py]: hmm, I bet this topic is definitely not resultsdb_style like.

File: resultsdbupdater/consumer.py (hunk 1 above)
Comment (thrix) [reply resultsdbupdater/consumer.py]: +1 , rpmdiff and covscan should be in the "resultsdb" style

File: tests/fake_messages/tps_message.json [hunk 4]
```diff
@@ -1,32 +1,60 @@
...
+      "ci_type": "ci-tps",
+      "content-length": "929",
+      "expires": "0",
+      "brew_task_id": "14645871",
+      "JMS_AMQP_MESSAGE_FORMAT": "0",
+      "JMS_AMQP_NATIVE": "false",
+      "destination": "/topic/VirtualTopic.eng.tps",
+      "component": "golang-1.9.2-4.bz1505967.el7",
+      "priority": "4",
+      "message-id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-46295-1510954431290-2:478022:0:0:1",
```
Comment (thrix) [inline tests/fake_messages/tps_message.json]: could we remove this internal name pls?

File: tests/fake_messages/tps_message.json [hunk 5]
```diff
@@ -1,32 +1,60 @@
...
+    "ci_type": "ci-tps",
+    "content-length": "929",
+    "expires": "0",
+    "brew_task_id": "14645871",
+    "JMS_AMQP_MESSAGE_FORMAT": "0",
+    "JMS_AMQP_NATIVE": "false",
+    "destination": "/topic/VirtualTopic.eng.tps",
+    "component": "golang-1.9.2-4.bz1505967.el7",
+    "priority": "4",
+    "message-id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-46295-1510954431290-2:478022:0:0:1",
```
Comment (thrix) [inline tests/fake_messages/tps_message.json]: could we remove this internal name pls?

File: tests/fake_messages/tps_message.json (hunk 4 above)
Comment (mprahl) [reply tests/fake_messages/tps_message.json]: Yup missed that one. Sorry.

**Comment by thrix** [comment]
There are still some internal names used in the tests (for tps at least). Please set the to something else.

File: resultsdbupdater/consumer.py (hunk 3 above)
Comment (mprahl) [reply resultsdbupdater/consumer.py]: It is, but I do need to add additional filtering to make sure it is for the AMI test results.

**Comment by mprahl** [comment]
Thanks for the review. I'm ready for round two.

Note: The unit tests work and pass, but I'll rewrite the unit tests in a separate PR since they aren't super helpful as they are now.

File: tests/fake_messages/message.json (hunk 2 above)
Comment (ralphbean) [reply tests/fake_messages/message.json]: FYI - @mprahl presented evidence to the contrary in IRC.

**Comment by thrix** [comment]
No addtional comments, seems to be fine.

**Pull request merged**